client/state/ui/selectors.js

```diff
@@ -6,7 +6,7 @@
  */
 export function getSelectedSite( state ) {
 	if ( ! state.ui.selectedSiteId ) {
-		return null;
+		return false;
```

Comment:
>I wonder if there was a reason to return `null` instead of `false` here.

`null` is the more "semantically accurate" value to return when there's no currently selected site. I'm not sure why `false` would ever be expected as a return value. That `false` results in `undefined` for accessing properties seems like a quirk that we shouldn't be relying on. Instead, we should always be guarding against the site not being set.